var/spack/repos/builtin/packages/r-fasterize/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class RFasterize(RPackage):
+    """Provides drop-in replacement for rasterize() from the package raster."""
+
+    homepage = "https://cran.r-project.org/package=fasterize"
+    url      = "https://cran.r-project.org/src/contrib/fasterize_1.0.3.tar.gz"
```

Comment:
Needs a `list_url`: https://spack.readthedocs.io/en/latest/build_systems/rpackage.html#list-url

Also, you should use `cloud` over `cran` for mirrors

Reply:
Can you use cloud for the homepage and url too?